site/docs/v1/tech/apis/identity-providers/openid-connect.adoc

```diff
@@ -23,6 +23,8 @@ FusionAuth will also leverage the `/userinfo` API that is part of the OpenID Con
 :idp_linking_strategy: pass:normal[`LinkByEmail`]
 :token_name: pass:normal[`refresh_token`]
 :return_text: pass:normal[external OpenID Connect provider, if such a token is provided,]
+:idp_long_lived_token_type: refresh
+:idp_refresh_additional_info_url: https://datatracker.ietf.org/doc/html/rfc6749
```

Comment:
We could link to the section - https://datatracker.ietf.org/doc/html/rfc6749#section-1.5

This is a broad document - so this might be too specific